### PR TITLE
helloworld-mdb 

### PR DESCRIPTION
**Pull request by spagop**
here my contribution with message-driven bean usecase using servlet 3.0 as client for sending several message to a queue.

### CODE REVIEW

File: quickstarts/helloworld-mdb/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+This example demonstrates the use of *JMS 1.1* and *EJB 3.1 Message-Driven Bean* in JBoss AS 7.
+
+The example can be deployed using Maven from the command line or from Eclipse using
+JBoss Tools.
+
+To set up Maven or JBoss Tools in Eclipse, refer to the _Getting Started Guide_.
+
+To deploy to JBoss AS 7, start JBoss AS 7 and type `mvn package jboss-as:deploy`.
+The application is deployed to <http://localhost:8080/jboss-as-helloworld-mdb/HelloWorldMDBServletClient>. You
+can read more details in the _Getting Started Guide_.
```
Comment (pmuir) [inline quickstarts/helloworld-mdb/README.md]: Given there is no tutorial for this, a more complete readme would be nice e.g. https://github.com/jbossas/quickstart/blob/master/servlet-filterlistener/README.md

Comment (spagop) [inline quickstarts/helloworld-mdb/README.md]: already done and tested again with the last nightly build of jboss as7 and it work fine.

**Comment by pmuir** [comment]
This is merged, right?

**Comment by spagop** [comment]
Yes, all changes are already merged.

**Pull request closed without merging**